android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/WorkerBinder.kt

```diff
@@ -194,6 +236,19 @@ public object WorkerBinder {
   }
 }
 
+/**
+ * Holds relevant information for [dispatcherAtBinder] [workerCoroutineWorker] and resulting
+ * [jobCoroutineWorker]
+ */
+public data class CoroutineContextInfo(
+  /** Dispatcher passed at WorkerBinder.bind call */
+  val dispatcherAtBinder: CoroutineDispatcher,
+  /** CoroutineContext defined at Worker */
+  val workerCoroutineWorker: CoroutineContext,
+  /** Effective CoroutineContext that will be used by the Worker binding */
+  val jobCoroutineWorker: CoroutineContext,
```

Comment:
Anything actually need to be held onto besides the jobCoroutineWorker? Feels like this may be confusing for consumers to understand the nuance of.

Reply:
I agree, let's keep just the jobCoroutineWorker then